vector/src/main/java/im/vector/activity/VectorRoomActivity.java

```diff
@@ -731,12 +731,22 @@ public boolean onEditorAction(TextView textView, int actionId, KeyEvent keyEvent
             }
         });
 
+        mRoom = mSession.getDataHandler().getRoom(roomId, false);
+
+        mEditText.initAutoCompletion(mSession, (null != mRoom) ? mRoom.getRoomId() : null);
+        mEditText.initAutoCompletionCommandLine(mSession , null);
```

Comment:
Maybe we could have only one public method 
`public void initAutoCompletions(@NonNull final Session session,  @Nullable Room room)`